code/backend/batch/utilities/parser/OutputParserTool.py

```diff
@@ -68,7 +68,11 @@ def parse(
                 {
                     "content": doc.get_markdown_url() + "\n\n\n" + doc.content,
                     "id": doc.id,
-                    "chunk_id": doc.chunk,
+                    "chunk_id": (
+                        re.findall(r"\d+", doc.chunk_id)[-1]
```

Comment:
Is `re.findall(r"\d+", doc.chunk_id)` guaranteed to have at least one item?

Reply:
So with IV, chunk_id will be in the form of "guid_page_1". 1 here is the chunk, so `re.findall(r"\d+", doc.chunk_id)` should return the last digit